test/sales_engine_test.rb

```diff
@@ -91,9 +90,46 @@ def test_it_passes_parsed_item_data_to_item_repository
     assert_equal '31163', engine.item_repository.items[6].unit_price
   end
 
+  def test_it_has_a_customer_repository
+    # skip
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [141/80]<br>Space inside { missing.<br>Space inside } missing.